executor/cook/_version.py

```diff
@@ -1,4 +1,4 @@
 # This file is read by setup.py to obtain the version.
 # Be aware that changing the format may break the parsing logic.
 
-__version__ = "0.1.2.dev0"
+__version__ = "0.1.3.dev0"
```

Comment:
What does the "dev0" mean? Do we need it?

Reply:
`devN` is the python equivalent of `SNAPSHOT` in versions. https://www.python.org/dev/peps/pep-0440/#version-scheme